lib/network/modules/components/nodes/util/NodeBase.js

```diff
@@ -89,6 +90,23 @@ class NodeBase {
       }
     }
   }
+
+
+  /**
+   * Determine if the shape of a node needs to be recalculated.
+   *
+   * @protected
+   */
+  needRefresh(selected, hover) {
```

Comment:
That was my initial thought too. However, it's not strictly `private` but `protected`, in the sense that it is called from derived classes. So I wasn't 100% sure to make it look like a private method.

Are there any guidelines for this?